### DISCUSSION TITLE
OMEdit - Generate Debug Symbols

### DISCUSSION THREAD

**Discussion by MatteoSchiavon** [OpenModelica/OpenModelica#13322] [General]
In the "FMI" toolbox, when I activate the "Generate Debug Symbols", is the OpenModelicaCompiler able to generate debug symbols (.pdb file) outside the result executable/library? Or is it somehow embedded in the target result?

**Comment by adrpo** [comment]
We don't generate PDBs as we are not using the MSVC compiler, we just compile with `clang` or `gcc` with `-g`.

However, it seems possible to generate PDBs with clang (not sure about pdbs):
https://stackoverflow.com/questions/74416539/clang-14-does-not-generate-pdb-file
Of course, this needs to be done manually by unzipping the FMU, changing the flags in cmake and rerun the build.
